readme.md

```diff
@@ -1,163 +0,0 @@
-# electron-packager
```

Comment:
My concern with having the readme be a symlink is that IIRC symlinks do not work on Windows.